language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/AbstractCodeActionTest.java

```diff
@@ -479,4 +483,31 @@ private boolean validateExtractCmd(JsonObject actualCommand, JsonArray actualArg
         actualCommand.add("arguments", newArgs);
         return false;
     }
+
+    private static final PrintStream printStream = System.out;
+
+    private boolean validateCreateConfigTomlCommand(JsonObject actualCommand, JsonArray actualArgs, JsonArray expArgs,
+                                                                                      Path sourceRoot) {
+        String actualFilePath = actualArgs.get(0).getAsString().replace(sourceRoot.toString(), "");
+        printStream.println(actualFilePath);
+        if (actualFilePath.startsWith("/") || actualFilePath.startsWith("\\")) {
+            actualFilePath = actualFilePath.substring(1);
+        }
+        String expectedFilePath = expArgs.get(0).getAsString();
+
+        String actualTextEdit = actualArgs.get(1).getAsString();
+        printStream.println(actualTextEdit);
+
+        String expectedTextEdit = expArgs.get(1).getAsString();
```

Comment:
How can we ensure that we get `expArgs` > 2?

Reply:
We check the arg size in line 442.